### DISCUSSION TITLE
Paru package in the aur is out of date ?

### DISCUSSION THREAD

**Discussion by MrDisguised** [Morganamilo/paru#974] [Q&A]
Hi !! I want to use paru. I want to change my AUR helper from yay to paru . The [paru package in the aur](https://aur.archlinux.org/packages/paru) says that it is out of date so I wanted to know which package should I install. (`paru` , `paru-bin` or `paru-git` ?)

**Comment by parkerlreed** [comment]
`paru-bin` is officially supported and is up to date.

**Reply by batot1** [reply]
With is it difference between peru and paru-bin? Can you explain?

**Reply by batot1** [reply]
And the second question, why are you releasing 2 versions of a few "paru" and "paru-bin" in parallel?

**Reply by parkerlreed** [reply]
paru is for compiling the release build

paru-bin is for the precompiled build from releases (as rust can take a long time to build on lower powered hardware)

paru-git is compiling from master

**Reply by somebody1234** [reply]
@batot1 note that the `x` + `x-bin` + `x-git` is an AUR-wide naming convention

**Comment by somebody1234** [comment]
@Morganamilo the [`paru`](https://aur.archlinux.org/packages/paru) package is flagged as out of date, might want to bump anyway, or unflag?

**Reply by eclairevoyant** [reply]
It's not out of date, but people lose their minds when the version number doesn't match the latest release, so I assume that's why she left the flag on.

**Reply by codewiz** [reply]
Could someone please explain why both paru and paru-bin exist in AUR?

Both these packages are owned by the upstream maintainer, their descriptions are identical, but the versions differ. This is why so many users are confused and keep asking for clarification.

**Reply by eclairevoyant** [reply]
@codewiz read the arch wiki

**Reply by codewiz** [reply]
Sorry @eclairevoyant, which wiki page should I read?

This one links to [paru](https://aur.archlinux.org/packages/paru): https://wiki.archlinux.org/title/AUR_helpers
I was unable to find a single reference to paru-bin in the Arch Wiki.

**Reply by somebody1234** [reply]
@codewiz https://unix.stackexchange.com/questions/543013/whats-the-difference-between-pkg-pkg-bin-and-pkg-git-on-the-aur

also here: https://wiki.archlinux.org/title/AUR_submission_guidelines#Rules_of_submission
(ctrl-f, `-bin`)
